lib/pgBackRest/Common/Ini.pm

```diff
@@ -353,7 +360,7 @@ sub iniParse
         logDebugParam
         (
             __PACKAGE__ . '::iniParse', \@_,
-            {name => 'strContent', trace => true},
+            {name => 'strContent', required => false, trace => true},
```

Comment:
In this case the parameter must always be passed, but it could be null.  This is because and openRead() with bIgnoreMissing set will return undef.  It's a fine distinction, I know.